CIP-DRep-Credential/README.md

```diff
@@ -0,0 +1,393 @@
+
+This CIP addresses these gaps by proposing an **on-chain** credentialing framework that leverages **off-chain** metadata anchors (per **CIP-119** for extended DRep profiles and **CIP-108** where proposal data is relevant). As a result:
+
+- **DReps** can mint concise on-chain credentials referencing richer off-chain information, making their profiles discoverable and verifiable within the ledger.
+- **Delegators** gain clearer insights into DReps’ qualifications, roles, endorsements, and historical voting behavior directly at the wallet or analytics-tool level—enabling more informed delegation choices.
+- **Community stakeholders** can independently endorse DReps on-chain, establishing verifiable social proof that strengthens the network’s trust fabric.
+
+By integrating these elements, this CIP fosters a more transparent, data-driven governance environment—ultimately encouraging broader participation and deeper accountability across Cardano’s evolving governance ecosystem.
+
+## Specification
+
+> "**We see categories as the best available formal conceptual tool for bridging those multiple worlds that exist in the large scale engineering of practical, robust, evolving systems.**"  
+> — Joseph A. Goguen (1997)
+
+This CIP proposes three **governance-oriented NFTs**—the **DRep Credential**, **Ballot Note**, and **Endorsement**—each referencing **CIP-1694** governance IDs (DReps, proposals) and optionally linking to extended off-chain data (via **CIP-119** or **CIP-108**). On-chain fields are stored within a **CIP-68** datum to keep records succinct yet verifiable, while token names follow **CIP-67** labeling.
+
+### 1. Relationship to Existing Standards
+
+- **CIP-1694 (On-Chain Governance)**
+
+  - Defines DRep IDs (blake2b-224 hash) and proposal IDs (transaction hash + index).
+
+- **CIP-68 (Datum-Based NFTs)**
+
+  - Enables storing metadata in a Plutus datum.
+  - This proposal uses CIP-68 for on-chain governance data (versioning, optional updatability).
+
+- **CIP-67 (Asset Name Labeling)**
+
+  - Enforces a 4-byte label scheme so wallets/explorers can recognize governance tokens.
+
+- **CIP-119 / CIP-108 (Off-Chain Anchors)**
+  - CIP-119: Off-chain JSON for extended DRep profiles.
+  - CIP-108: Off-chain metadata for proposals or related governance documents.
+
+**Note:** The technical details of each CIP (hashing, labeling, datum layouts) are **not** restated here. Implementers must consult each CIP’s original documentation.
+
+### 2. Identifiers (DRep & Proposal)
+
+1. **DRep ID**
+
+   - A **28-byte (56-hex)** blake2b-224 hash of the stake key or script credential per CIP-1694.
```

Comment:
```suggestion
   - A **28-byte (56-hex)** blake2b-224 hash of the DRep key or script credential per CIP-1694.
```

Reply:
Throughout youve referenced identifying DRep via stake credentials
I believe this to be a mistake, as DReps are identified on Cardano by DRep credentials

Reply:
@Ryun1 Yes this is correct. It has to be the DRep credential but we are also considering identifying through a second credential that either we issue, ADA handle issues, or something else in order to allow for endorsements. We want the endorsement system to be something that can not change. Once someone has made an endorsement through ADA handle or our platform, then the user can decide to accept the endorsement or deny it from displaying on their profile. 
We also want the endorser to be able to revoke endorsements. There is still a little confusion around exactly how we are going to do this part but we may have been thinking stake key originally. Idk. @Alpine-Oracle @dagwell and the @Adahandle people are working on this with me.